lib/ui/gallery_app_bar_widget.dart

```diff
@@ -23,6 +23,7 @@ enum GalleryAppBarType {
   shared_collection,
   collection,
   search_results,
+  incoming_collection,
```

Comment:
Perhaps you could rename `collection` to `owned_collection` for clarity? Also, use `shared_collection` instead of `incoming_collection`? I don't think it's being used anywhere right now.

Reply:
Yes, `shared_collection` is only referred at once place but we are not setting it.  Will remove this particular reference of `share_collection` in the code here and reserve 'shared_collection` usage for incoming collections only.

https://github.com/ente-io/frame/blob/a0cbf3141e2d58f46605b50280225e817a24b268/lib/ui/gallery_app_bar_widget.dart#L197